Features/mobile menu javascript

Wahaj-Ali opened the pull request:
In this branch
- implemented JavaScript for the "hamburger icon" in the mobile version of the desktop. 
- created scripts.js to add JavaScript code.
- Restyled the mobile version to adjust according to the changes made.

utsargo committed:
Implemented mobile menu without transitions

Wahaj-Ali committed:
In this branch, implemented the JavaScript for the mobile menu

in this commit
- re-implemented JavaScript for the hamburger icon in the mobile-version of portfolio
- restyled the portfolio to adjust according to the changes made.
- added script.js to the repo.

Wahaj-Ali committed:
updated styles.css

in this commit
- removed linter errors

Wahaj-Ali committed:
Update styles.css

in this commit
- removed further linter errors

Wahaj-Ali committed:
Update linters.yml

in this commit
- added the JavaScript + HTML + CSS linter.

Wahaj-Ali committed:
removed linter errors

in this commit
- removed linter errors from styles.css and script.js

Wahaj-Ali committed:
updated menu links text

in this commit
- edit the font size and font color for the menu links for mobile version

Wahaj-Ali committed:
Update styles.css

in this commit
- remove linter errors

ha-manel commented on index.html:
```diff
@@ -13,139 +13,154 @@
+        <nav class="desktop-nav">
+            <a id="name" href=""><img class="mylogo" src="attributes/mylogo.png" alt="logo" /></a>
+            <ul class="main-menu">
+                <li class="nav-item"><a class="header-btn" href="">Portfolio</a></li>
```
- [ ] This link is supposed to take us to the works section. Kindly update the `href` accordingly 👍

ha-manel commented on index.html:
```diff
@@ -13,139 +13,154 @@
-            <a class="header_btn" href=""><img id="mail" src="attributes/mailicon.png" alt="mailicon" /></a>
+    <header>
+        <nav class="desktop-nav">
+            <a id="name" href=""><img class="mylogo" src="attributes/mylogo.png" alt="logo" /></a>
```
- [ ] Kindly hide the logo from inside the mobile menu to match the design in Figma. Also, kindly add `border-bottom` to all links to match the design 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/201729575-8a45e46d-4204-41ce-bdb1-93a5bed1c72d.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/201729609-af1c3a5e-c814-4cbf-b948-ae3831d5811f.png)

ha-manel commented on index.html:
```diff
@@ -13,139 +13,154 @@
-            <a class="header_btn" href="#full_name">Contact</a>
-            <a class="header_btn" href=""><img id="mail" src="attributes/mailicon.png" alt="mailicon" /></a>
+    <header>
+        <nav class="desktop-nav">
```
[Optional]
- [ ] It's better if we can't scroll when we open the mobile menu 👍 

![Animation](https://user-images.githubusercontent.com/50721479/201730079-736e4d62-b781-4db8-b455-4fe26b320bba.gif)

ha-manel reviewed (changes requested):
# STATUS: REQUIRED CHANGES ♻️

## Hi Team👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  

<img src="https://media3.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif"/>

## Highlights:  
* All tests passed ✔️
* Professional README file ✔️

## Required Changes:
_Kindly check the comments below_

## Optional:
_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Wahaj-Ali committed:
Update index.html

in this commit
- edit href for portfolio button to access the works section

Wahaj-Ali committed:
Made required changes

in this commit
- remove logo when in the menu of mobile version as required by the reviewer.

Wahaj-Ali committed:
Update script.js

in this commit
- removed linter errors

prolajumokeoni reviewed (approved):
Hi @Wahaj-Ali 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀   

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
-  Kindly make sure that for [pair programming](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#work-and-submission-mode) projects, both partners have commits, currently, there are no commits from your partner 👍🏿 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
